README.md

```diff
@@ -1,63 +1,73 @@
 [![Build Status](https://travis-ci.org/ethereum/browser-solidity.svg?branch=master)](https://travis-ci.org/ethereum/browser-solidity)
 
 # Remix
-
 Remix (aka. Browser-Solidity) is a browser-based Solidity compiler and IDE.
 
 Visit [https://remix.ethereum.org](https://remix.ethereum.org) to use;
 it will always deliver the latest version.
 
+
 ## Offline Usage
+The `gh-pages` branch always has the latest stable build of Browser-Solidity.  
```

Comment:
Oh sure - no problem - that was an unintentional change - not even aware i changed it, but i guess i did.